nix/scripts/purge.sh

```diff
@@ -50,16 +49,8 @@ nix_purge_darwin_multi_user_volumes() {
         local pid=$(lsof +D /nix | awk 'NR==2{print $2}')
         if [ -n "$pid" ]; then
```

Comment:
Double square brackets should be used:

>The single bracket is a built-in command that’s older than the double bracket.

https://www.baeldung.com/linux/bash-single-vs-double-brackets

Reply:
resolved in https://github.com/status-im/status-mobile/pull/18656/commits/4d46f1450a6b1fb08f6cdc4dd345cd2d152acf7a